ios/Sample/sampleTests/sampleTests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  sampleTests.swift
+//  sampleTests
+//
+//  Created by Tobias Heine on 16.04.19.
+//  Copyright © 2019 Novoda. All rights reserved.
+//
+
+import XCTest
+@testable import sample
+import main
+
+
+class sampleTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testSha256Request() {
```

Comment:
This is the swift port of `commonTest/Sha256Test`

Reply:
Shouldn't we move the `commonTest/Sha256Test` into the `jvmTest` as well?

Reply:
No, the idea is that you provide tests in the common module and the plugin executes the tests for each platform. Just that the ios support is still missing for this. Once this has been added we instead want to remove this swift test.

Reply:
Oh I see, awesome!